keps/1618-optional-gc-of-workloads/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Optional garbage collection of finished and deactivated Workloads
+kep-number: 1618
+authors:
+  - "@mwysokin"
+  - "@mykysha"
+status: implementable
+creation-date: 2024-07-31
+reviewers:
+  - "@denkensk"
+  - "@mimowo"
+  - "@PBundyra"
+  - "@tenzen-y"
+  - "@mbobrovskyi"
+approvers:
+  - "@mimowo"
+  - "@tenzen-y"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
Since the API is beta we need to either call it beta, or introduce a feature gate in alpha first. I would suggest a feature gate.

cc @mwysokin @tenzen-y wdyt?

Reply:
+1 on feature gate.

Reply:
Have added a feature-gate here